src/tiles.rs

```diff
@@ -151,6 +151,13 @@ impl<Pane> Tiles<Pane> {
     ///
     /// All removed tiles are returned in unspecified order.
     pub fn remove_recursively(&mut self, id: TileId) -> Vec<Tile<Pane>> {
+        // Remove the top tile_id from its parent
```

Comment:
This makes this function different from how `remove` acts.

We should at least document that, or make them act the same.

Perhaps the loop below should just be its own function call instead, `pub fn remove_from_parent(&mut self, id: TileId)` ?